content/src/content/jcr_root/apps/core/wcm/components/title/v2/title/_cq_design_dialog/.content.xml

```diff
@@ -128,6 +128,15 @@
                                                 value="h6"/>
                                         </items>
                                     </type>
+                                    <disableLink
+                                        jcr:primaryType="nt:unstructured"
+                                        sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
+                                        fieldDescription="Check to disable support for user defined ID's."
+                                        name="./disableId"
```

Comment:
I would favor a positive property name instead and let the name be a description, not an action: `idEnabled`.